geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/EmbeddedKafka.scala

```diff
@@ -33,6 +33,11 @@ class EmbeddedKafka extends Closeable {
 
   val brokers = s"127.0.0.1:${server.socketServer.boundPort()}"
 
+  // for kafka 1.0.0:
+  // import org.apache.kafka.common.network.ListenerName
+  // import org.apache.kafka.common.security.auth.SecurityProtocol
+  // val brokers = s"127.0.0.1:${server.socketServer.boundPort(ListenerName.forSecurityProtocol(SecurityProtocol.PLAINTEXT))}"
+
```

Comment:
Hmm?  The test has to change with 1.0.0?!

Reply:
yeah, but in general you wouldn't ever need to compile with other versions of kafka - i just left it in there for the future

Reply:
Roger that.